usaspending_api/api_contracts/contracts/search/AdvancedSearch.md

```diff
@@ -348,6 +347,22 @@ List of table columns
 + `lower_bound` (optional, number)
 + `upper_bound`: 1000000 (optional, number)
 
+### TASCodeObject (object)
++ `ata` (optional, string, nullable)
+    Allocation Transfer Agency Identifier - three characters
++ `aid` (required, string)
+    Agency Identifier - three characters
++ `bpoa` (optional, string, nullable)
+    Beginning Period of Availability - four digits
++ `epoa` (optional, string, nullable)
+    Ending Period of Availability - four digits
++ `a` (optional, string, nullable)
+    Availability Type Code - X or null
++ `main` (required, string)
```

Comment:
Question: is `main` used for Federal Account filtering?

Reply:
Yes.  Federal Account is `aid` and `main` (and technically `sub`, but my understanding is that we're removing that from the front end).